docs/components/SideNav.tsx

```diff
@@ -18,10 +18,8 @@ export const SideNav: React.FC<{ className?: string }> = ({ className }) => {
       {/* using css fallback properties for cross-browser compatibility  */}
       {/* read more: https://modernweb.com/using-css-fallback-properties-for-better-cross-browser-compatibility/ */}
       <style jsx>
-        {" "}
         {`
           .sidebar-container {
-            height: calc(100vh - var(--top-nav-height) - 4px);
```

Comment:
I wonder why was this needed resulting two entries the linter is complaining about ? There was no comment.

Reply:
please dont remove this, there is a comment above this. `100vh` is a fallback of `100dvh`